Podcast/UserDetailViewController.swift

```diff
@@ -0,0 +1,413 @@
+//
+//  UserDetailViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 3/7/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+final class UserDetailViewController: ViewController {
```

Comment:
Why final? for performance?

Reply:
if yes, make a comment saying that in case anyone wants to subclass in the future they know that is the reason

Reply:
yeah it uses static dispatching this way. Also it won't let them subclass, if they want to they can just remove final. It's good practice to make it final if you're not going to subclass because it can dispatch statically